docs/conf.py

```diff
@@ -68,7 +68,7 @@
 # a list of builtin themes.
 #
 html_theme = "pydata_sphinx_theme"
-html_logo = "logo.svg"
+html_logo = "_static/logo.svg"
```

Comment:
e.g.: here we're adding `_static` (see comment below):

https://github.com/pydata/pydata-sphinx-theme/pull/1132/files#r1094554022

Reply:
So this is were things become messy, the `logo` (or `logo_url`) variable in the context is only the stem of this file. The complete path is only useful for sphinx to copy the file in `_static`. 

BUT I need to check again (it's working on my local computer) it's driving me crazy